mcs/class/corlib/System.IO/MonoIO.cs

```diff
@@ -169,32 +169,80 @@ public static Exception GetException (MonoIOError error)
 		// directory methods
 
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
-		public extern static bool CreateDirectory (string path, out MonoIOError error);
+		public extern static bool CreateDirectory (IntPtr path, out MonoIOError error);
```

Comment:
`private`?

Actually why not `private unsafe static bool CreateDirectory (char *path, out MonoIOError error);` ?

Reply:
No particular reason; Ludovic used `IntPtr` in his example and I just aped it. I guess more types couldn’t hurt.